### PR TITLE
Add TropicalGames

### PR DESCRIPTION
**Pull request by Alessandro2012012**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by minepodcraft** [comment]
i do not linke this server it is a scammer

**Comment by OreoBladesword** [comment]
This server is very unfriendly to new players.
i was a new player and they disrespected me.
the co owner laify is the biggest problem!

File: minecraft_servers/TropicalGames/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "TropicalGames",
```
Comment (DoJapHD) [inline minecraft_servers/TropicalGames/manifest.json:2]: server_name has to be lowercase (folder name too!)

File: minecraft_servers/TropicalGames/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "nice_name": "TropicalGames",
+  "direct_ip": "TropicalGames.de",
+   "server_wildcards":[
+     "premium.TropicalGames.de",
+     "%.TropicalGames.de"
+  ],
+  "social": {
+    "web": "https://TropicalGames.de",
+    "web_shop": "https://TropicalGames.de/store",
```
Comment (DoJapHD) [inline minecraft_servers/TropicalGames/manifest.json:11]: /store leads to 404

File: minecraft_servers/TropicalGames/manifest.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "TropicalGames",
+  "nice_name": "TropicalGames",
+  "direct_ip": "TropicalGames.de",
+   "server_wildcards":[
+     "premium.TropicalGames.de",
```
Comment (DoJapHD) [inline minecraft_servers/TropicalGames/manifest.json:6]: Redundant, %.tropicalgames already implements premium.tropicalgames

File: minecraft_servers/TropicalGames/manifest.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "TropicalGames",
+  "nice_name": "TropicalGames",
+  "direct_ip": "TropicalGames.de",
```
Comment (DoJapHD) [inline minecraft_servers/TropicalGames/manifest.json:4]: IP please in lowercase (also in wildcards)

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Didn't answer, reopen if still needed

**Pull request closed without merging**